__tests__/BottomTabNav.test.js

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import { NavigationContainer } from "@react-navigation/native";
+import { render, screen, fireEvent } from '@testing-library/react-native';
+
+import AppNavigator from '../src/Shared/Navigation/BottomTabNavigator'
+
+describe('Testing react navigation', () => {
+  test('clicking on the bookmarks button takes you to the bookmarks screen', async () => {
+    const component = (
+      <NavigationContainer>
+        <AppNavigator />
+      </NavigationContainer>
+    );
+
+    render(component);
+    const homeScreen = await screen.queryByText('HomeScreen');
```

Comment:
All these screen queries will fail once the placeholder screens are replaced; how should we handle that? Correct these tests once they are replaced?

Reply:
yes we will have to change some of the tests as the app is filled out, the tests will need to change as we start putting the components together.